charts/shoot-core/charts/kube-proxy/templates/kube-proxy-daemonset.yaml

```diff
@@ -42,6 +42,18 @@ spec:
         - net.ipv6.conf.all.disable_ipv6=1
         securityContext:
           privileged: true
+{{- end }}
+{{- if and .Values.enableCleanup (semverCompare "> 1.14.0" .Values.kubernetesVersion) }}
```

Comment:
You mean rebuilding the whole ```ipvs``` and ```iptable``` cleanup logic in a bash/shell script, which might be error-prone or calling ```kube-proxy --cleanup``` and waiting for a certain amount of time to make sure the job has finished?

Reply:
I really hoped that the cleanup logic could be rebuilt in a shell script. I'm not sure though how much effort it would be.

Otherwise we should prohibit a 1.14.1 cluster from switching the `Kube-Proxy` mode, like @rfranzke already mentioned.